### PR TITLE
TheNextWeb-Clone Project 2

### PR DESCRIPTION
**Pull request by diamond-nicholas**
##in this project, we tried to clone the next web page and make it responsive for all screens
##we used the html5 and css3 to clone the webpage while we applied some css styles including css grid, float, and flex.

### CODE REVIEW

**Comment by descholar-ceo** [comment]
# This Pull Request is under review by [descholar-ceo](https://github.com/descholar-ceo) :eyes: :eyes: 
## Please wait a bit ...

File: README.md [hunk 1]
```diff
@@ -1 +1,69 @@
...
+
+## Built With
+
+- HTML5
+- CSS3#
+
+
+## Live Demo
+
+[Live Demo Link]()
```
Comment (descholar-ceo) [inline README.md]: The live demo link is empty

File: README.md [hunk 2]
```diff
@@ -1 +1,69 @@
...
+### Desktop version
+
+![screenshot](./)
+
+Additional description about the project and its features.
+
+## Built With
+
+- HTML5
+- CSS3#
```
Comment (descholar-ceo) [inline README.md]: It should be CSS3 not ~CSS3#~

File: README.md [hunk 3]
```diff
@@ -1 +1,69 @@
-# TheNextWeb-Clone
\ No newline at end of file
+# TheNextWeb-Clone
+
+# Project 2: Building with Responsive Design
+
+### Desktop version
+
+![screenshot](./)
```
Comment (descholar-ceo) [inline README.md]: Add a relevant screenshot

**Review by descholar-ceo — CHANGES_REQUESTED** [review]
# Status: Changes Required :vertical_traffic_light: :warning: 
## Hello @diamond-nicholas and @charlyeneh   :handshake: :handshake: :handshake: 

:+1: I liked that you have put in effort to make this project this far,
:+1: You have used sementic HTML tags, that is worth appreciation, keep it up :muscle: 
:+1: You gone mobile first approach in your CSS codes, that is commendable. Kudos :muscle: 
#### :warning: However we need to commit even more effort in order to get this project to the approval. See the requested change below :point_down: :point_down: :point_down: 
### Note: All of requested changes which are marked as [Required] are mandatory to be implemented in order to get this project approved, others which are marked as [Optional] are nice to have in your project to make it outstanding

# [Required Changes] :red_circle: 

### General
- [ ] Fix the lincense link from the readme file, it's not working
- [ ] It is better to remove the `deployment` section from your readme file, currently you have put `Deploy on gh pages for testing purposes` this seems to not be clear.
- [ ] This [live demo link](https://github.com/diamond-nicholas/TheNextWeb-Clone/blob/workspace) from your readme file doesn't work, please fix it.
- [ ] Add the project's description in your readme file, describe your project, it makes your repo look very professional.
- [ ] Fix the screenshot from the readme file, it's broken
- [ ] In `Built with` section of your readme file, instead of writting `CSS3#` it should be `CSS3`

### UI/UX
- [ ] Fix the `navbar` to make it look like the original

| Your clone | Original page |
| --- | --- |
| Screenshot from 2020-12-10 01-48-12 | Screenshot from 2020-12-10 01-50-14|

:point_right: Could you see the difference :question: :question: :question: 

- [ ] Fix the `Latest news` section to make it look like the original page

| Your clone | Original page |
| --- | --- |
|Screenshot from 2020-12-10 01-51-53|Screenshot from 2020-12-10 01-52-33 |

:point_right: Could you see the difference :question: :question: :question: 

- [ ] Fix the `Latest funding rounds on Index.co >` to make it look like the one found on the original page 

| Your clone | Original page |
| --- | --- |
|Screenshot from 2020-12-10 01-53-36|Screenshot from 2020-12-10 01-54-16|

:point_right: Could you see the difference :question: :question: :question: 

- [ ] Fix the `Categories` section to make it look like the origin 

| Your clone | Original page |
| --- | --- |
|Screenshot from 2020-12-10 01-55-15|Screenshot from 2020-12-10 01-55-56|

:point_right: Could you see the difference :question: :question: :question: 

- [ ] Fix the `Latest Deals >` to make it look like the original

| Your clone | Original page |
| --- | --- |
|Screenshot from 2020-12-10 01-57-08|Screenshot from 2020-12-10 01-58-25|

:point_right: Could you see the difference :question: :question: :question: 
- [ ] The icons in the footer should be horizontally and vertically alligned

### Codes
- [ ] Your CSS codes should be DRY (Don't Repeat Yourself)=> You are repeating yourselves and that is bad because your codebase becomes very big and hard to maintain or debug. For example: You are repeating `display: none;` for 11 times, `display: block;` for 5 times, `display: flex;`, `display: grid;` for 3 times, `grid-template-columns: auto auto auto;` for 5 times => t=you have a lot of peoperties in your CSS codes which are being repeating so frequently, as advice reduce the repeating times of properties by using utility classes or grouping. Refer to [this article to know more about using best parctices in CSS](https://speckyboy.com/good-bad-css-practices/)
- [ ] Make sure that there are no remaining unfixed linters errors before you re-submit for code review
- [ ] Implement all changes I suggested in inline comments in your codes

# [Optional changes] :yellow_circle: 
:warning: Your commit messages should be in imperative mode or present tense, _i.e.:_ you should have written `Fix linter errors` instead of writing `fixing linter errors`, refer [to this article for more details on how to write a good && professional commit message](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8)

___
_If anything seems to be unclear to you, reachout and ask anything by commenting on this Pull Request and mention me as [descholar-ceo](https://github.com/descholar-ceo), I will receive a notification and will respond._

[GitHub](https://github.com/descholar-ceo) | [LinkedIn](https://linkedin.com/in/mugirase-emmanuel) | [Twitter](https://twitter.com/descholar3)

_Reviewed by [Mugirase Emmanuel (descholar)](https://github.com/descholar-ceo)_

**Comment by mimipeshy** [comment]
## STATUS: CHANGES REQUIRED 🔴 ♻️ 

Hello @diamond-nicholas and @charlyeneh 👋 👋 

✔️  Your project looks really nice, it almost looks like the original one
✔️  Good job on working on the previous TSE comments

However, before marking your project as complete, check out the comments below  as left by the previous TSE⬇️ 

- [ ] Fix the license link from the readme file, it's not working. It throws a 404 error thus making your repo unprofessional. Kindly fix that.

- [ ] It is better to remove the deployment section from your readme file, currently, you have put Deploy on gh pages for testing purposes this seems to not be clear.

- [ ] Your CSS codes should be DRY (Don't Repeat Yourself)=> currently you are repeating  `grid-template-columns: auto auto auto;` 4 times. Kindly correct this

- [ ] Make sure that there are no remaining unfixed linters errors before you re-submit for code review. Refer to this [link](https://github.com/microverseinc/linters-config/tree/master/html-css) on how to setup linter files and how to solve them.


That's all. 😄 , kindy submit another review when you are done

Regards
Peris Ndanu
Feel free to reach out on slack if you have any questions

**Review by arslanbisharat — APPROVED** [review]
# Approved :smile: 

Hello Developer :wave:

I just reviewed your project and I am very happy to inform you that your project is approved.
Congratulations on achieving this milestone in your career. :clap:



_I am looking forward to seeing your next project._

Stay safe and give me a smile 😇

### Happy Coding
------------------------------------------------------------------------------------------------
**Regards,
Arslan Bisharat
slack: arslanbisharat**

**Pull request merged**